.fixtures.yml

```diff
@@ -3,6 +3,11 @@ fixtures:
     stdlib: 'https://github.com/puppetlabs/puppetlabs-stdlib'
     vcsrepo: 'https://github.com/puppetlabs/puppetlabs-vcsrepo'
     redis: 'https://github.com/arioch/puppet-redis'
+    python: 'https://github.com/stankevich/puppet-python'
+    epel:
+      repo: 'https://github.com/stahnma/puppet-module-epel.git'
+      ref: '1.2.2'
```

Comment:
please don't pin the dependency. For unit tests we want to test against the master branch of our dependencies to catch breaking changes before a release happens. We can test against concrete versions in acceptance tests.

Reply:
Thanks for the feedback! I've un-pinned the dependency.